speeduino/speeduino.ino

```diff
@@ -225,14 +223,12 @@ void loop()
 
     //***Perform sensor reads***
     //-----------------------------------------------------------------------------------------------------
-    readMAP();
-    
-    if (BIT_CHECK(LOOP_TIMER, BIT_TIMER_15HZ)) //Every 32 loops
+    ADC_sequencer();//All ADC reads are now combined into sequencer to ensure optimal loop speed without any blocking
+    BIT_CLEAR(TIMER_mask, BIT_TIMER_40HZ); //only ADC uses this at the moment
```

Comment:
This must be removed as it will constantly clear the flag and might make the loop miss the flag, check previous comment.